### PR TITLE
[docs] remove docs script in each package as we now have the multi-modules documentation

### PR DESCRIPTION
**Pull request by benoitf**
- [X] bump of version of typedoc is covered by CQ https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18832

follow up on https://github.com/theia-ide/theia/pull/4636 and what was discussed at DevMeeting last week

Change-Id: I0c47554e1a99e22c7b519c94afa7cfd9be1622a4
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

**Comment by akosyakov** [comment]
Thanks for clean up!

When it run `yarn docs` it produces a lot of errors for me now, although seem to succeed. Is it expected? 

Errors like:
```
Error: /workspace/theia/node_modules/@theia/plugin/src/theia.d.ts(3540)
 Duplicate string index signature.
Error: /workspace/theia/node_modules/@theia/plugin/src/theia.d.ts(5946)
 Duplicate string index signature.
Error: /workspace/theia/node_modules/@theia/plugin/src/theia.d.ts(6553)
 Duplicate string index signature.
Error: /workspace/theia/node_modules/@theia/plugin/src/theia.d.ts(7042)
 Duplicate string index signature.
Error: /workspace/theia/node_modules/monaco-languageclient/lib/monaco-converter.d.ts(0)
 Cannot find type definition file for 'monaco-editor-core/monaco'.
Error: /workspace/theia/node_modules/monaco-languageclient/lib/monaco-diagnostic-collection.d.ts(0)
 Cannot find type definition file for 'monaco-editor-core/monaco'.
Error: /workspace/theia/node_modules/monaco-languageclient/lib/monaco-languages.d.ts(0)
 Cannot find type definition file for 'monaco-editor-core/monaco'.
Error: /workspace/theia/node_modules/monaco-languageclient/lib/monaco-workspace.d.ts(0)
 Cannot find type definition file for 'monaco-editor-core/monaco'.
```

It was not the case before.

File: configs/typedoc.json [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
   "excludeExternals": true,
-  "includeDeclarations": true
+  "includeDeclarations": true,
+  "readme": "none",
+  "module": "commonjs",
+  "target": "es6",
+  "hideGenerator": true,
+  "external-modulemap": ".*\/packages\/([\\w\\-_]+)\/",
+  "name": "Theia TypeDoc",
+  "exclude": "**/+(dev-packages|examples|typings)/**/*.ts"
```
Comment (akosyakov) [inline configs/typedoc.json]: Could we add one more exclude to filter out spec files:
```json
"exclude": [
    "**/+(dev-packages|examples|typings)/**/*.ts",
    "**/*spec.ts"
  ]
```

Comment (benoitf) [reply configs/typedoc.json]: I'm not sure it works with this format of exclude ? but I may be wrong of course

Comment (akosyakov) [reply configs/typedoc.json]: It seems to be valid syntax: https://github.com/TypeStrong/typedoc/issues/170#issuecomment-388607358

Comment (benoitf) [reply configs/typedoc.json]: ok nice 👍

**Comment by benoitf** [comment]
added new exclude.

Now there are only errors about some missing typings but as we're excluding typings it may be related

**Comment by akosyakov** [comment]
> Now there are only errors about some missing typings but as we're excluding typings it may be related

right, looking at missing types they don't affect APIs only implementation, so it should be alright

**Pull request merged**